engine_wrapper.py

```diff
@@ -316,41 +330,36 @@ def get_stats(self, for_chat: bool = False) -> list[str]:
         can_index = self.move_commentary and self.move_commentary[-1]
         info: MOVE_INFO_TYPE = self.move_commentary[-1].copy() if can_index else {}
 
-        def to_readable_value(stat: str, info: MOVE_INFO_TYPE) -> str:
-            readable: dict[str, Callable[[Any], str]] = {"score": self.readable_score, "wdl": self.readable_wdl,
-                                                         "hashfull": lambda x: f"{round(x / 10, 1)}%",
-                                                         "nodes": self.readable_number,
-                                                         "nps": lambda x: f"{self.readable_number(x)}nps",
-                                                         "tbhits": self.readable_number,
-                                                         "cpuload": lambda x: f"{round(x / 10, 1)}%"}
-
-            def identity(x: Any) -> str:
-                return str(x)
-
-            return str(readable.get(stat, identity)(info[stat]))
-
-        def to_readable_key(stat: str) -> str:
+        def to_readable_key(stat: str, value: Any) -> str:
             readable = {"wdl": "winrate", "ponderpv": "PV", "nps": "speed", "score": "evaluation"}
             stat = readable.get(stat, stat)
+            if stat == "string" and value in ["Opening Book", "Lichess Opening Explorer (Masters)",
+                                              "Lichess Opening Explorer (Player)", "Lichess Opening Explorer (Lichess)",
+                                              "Lichess EGTB", "ChessDB EGTB", "Lichess Cloud Analysis", "ChessDB"]:
+                stat = "source"
             return stat.title()
```

Comment:
I think there's an easier way to do this without needing to list every possible source. In steps:
1. Prefix all of the `"string"` comments with `"lichess-bot-source:"`. For example, in line 668: `return chess.engine.PlayResult(move, None, {"string": "lichess-bot-source:Opening Book"})`
2. Rename this inner function `def to_readable_item()` with the same arguments.
3. The `if`-statement becomes `if stat == "string" and value.startswith("lichess-bot-source:"):`
4. If so, `stat = "source"` and `value = value.split(":", 1)[1]`
5. `return stat.title(), value`
6. Finally, `info = dict(to_readable_item(key, value) for (key, value) in info.items())`

Reply:
Done.